gnssanalysis/_version.py

```diff
@@ -394,7 +419,7 @@ def render_pep440(pieces):
     return rendered
 
 
-def render_pep440_branch(pieces):
+def render_pep440_branch(pieces: Dict[str, Any]) -> str:
     """TAG[[.dev0]+DISTANCE.gHEX[.dirty]] .
```

Comment:
Might be good to start with a description and then this
Also, One-line sphinx docstring (missing the description of input parameters, errors raised, what's returned, etc.: `:param` `:return`)

Reply:
file autogenerated by the versionner script. so won't be addressed